craco.config.js

```diff
@@ -1,4 +1,46 @@
 const CracoAlias = require('react-app-alias');
+const fs = require('fs');
+const path = require('path');
+const webpack = require('webpack');
+
+// Load the default config
+const defaultConfigPath = path.resolve(__dirname, 'src/config/config.default.json');
+const accessConfig = JSON.parse(fs.readFileSync(defaultConfigPath, 'utf8'));
+
+// Parse environment variables
+let ACCESS_TIME_LABELS_OVERRIDE = null;
+try {
+  ACCESS_TIME_LABELS_OVERRIDE = process.env.ACCESS_TIME_LABELS_OVERRIDE
```

Comment:
Instead of using environment variables to override the default config (like in https://github.com/discord/access/pull/194), can we read from the override config file like you had before using the define plugin below?

Reply:
The override config file path can be set by an environment variable (like before) so that we know whether or not to include it here.